### ISSUE TITLE
Add more accessibility options

### ISSUE THREAD

**Issue by marcellesilva** [florence-social/mastodon-fork#157]
I was looking at the https://colorid.es settings page and saw these accessibility options. I believe they are really helpful and could be implemented in the future.

![](https://i.imgur.com/ZorT1pw.png)

**Comment by clarfonthey** [comment]
So, to clarify:

1. Being able to configure how big emojis show up in places
2. Being able to enable/disable style options individually instead of as a whole

?

**Comment by marcellesilva** [comment]
Yes. I'm sorry for not describing the picture. I didn't know what terms to use since English is not my main language. I think font size could be added in this issue as well.

**Comment by clarfonthey** [comment]
That is okay, and thank you for filing the issue!
